portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,19 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>3030</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Mariano Shaar</h1>
+      <a href="https://github.com/shaargtz" class="link">
+        <img src="./images/icons/github.svg" alt="GitHub" class="icon">
```

Comment:
Make sure the alt text describes what the img is for, so that if you were using a screenreader you could tell. For these icons, probably 'Github link icon' and 'Linked link icon' or something similar would work.

Reply:
Done!